src/pykeen/nn/emb.py

```diff
@@ -1068,3 +1070,127 @@ def forward(
         if indices is not None:
             x = x[indices]
         return x
+
+
+class NodePieceRepresentation(RepresentationModule):
+    """
+    Basic implementation of node piece decomposition [galkin2021]_.
+
```

Comment:
request for fancy math documentation of what this is doing

Reply:
7e767bc 

this does not necessarily match the paper's description, but I found it more intuitive given that you may not have read the paper until then.

Reply:
Excellent. I strive for the PyKEEN documentation to be educational, not just a way to make stuff look good. @migalkin already did an excellent job of that in his paper 💯